src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -81,4 +82,50 @@ def get(self, request: Request, project: Project, event_id: str) -> Response:
                     ],
                 }
             )
+
+        release = Release.objects.get(project_id=project.id, version=release_version)
+        user_agent = release.user_agent
+
+        if not user_agent:
+            return Response(
+                {
+                    "errors": [
+                        SourceMapProcessingIssue(
+                            SourceMapProcessingIssue.MISSING_USER_AGENT
+                        ).get_api_context()
+                    ],
+                }
+            )
+
+        num_artifacts = release.count_artifacts()
+
+        if num_artifacts == 0:
+            return Response(
+                {
+                    "errors": [
+                        SourceMapProcessingIssue(
+                            SourceMapProcessingIssue.MISSING_SOURCEMAPS
+                        ).get_api_context()
+                    ],
+                }
+            )
+
+        exception = event.interfaces["exception"]
+        frame_list = exception.values[0].stacktrace.frames
```

Comment:
@roggenkemper So it's possible there could be two exceptions I think, does the CLI explain handle this the same way?

Reply:
I think so - https://github.com/getsentry/sentry-cli/blob/master/src/commands/sourcemaps/explain.rs#L376 , the CLI also looks at the values of the event.exception.values and takes the first one